samples/perf_client/perf_client.h

```diff
@@ -63,6 +64,7 @@ namespace client
     size_t max_writes_ahead = 0;
     size_t latency_rounds = 1;
     size_t generator_seed = 42u;
+    size_t transactions_per_s = 0;
```

Comment:
Can this be made to be ms. Second seems like a very high granularity?

Reply:
It's just used to set the interval, it's purely a UX concern. We mostly report tx/s rates, so this seems consistent with our existing practice/numbers?